docs/production-deployment/self-hosted-guide/archival.mdx

```diff
@@ -125,8 +125,8 @@ The Archival URI cannot be changed after the Namespace is created.
 Each Namespace supports only a single Archival URI, but each Namespace can use a different URI.
 A Namespace can safely switch Archival between `enabled` and `disabled` states as long as Archival is enabled at the cluster level.
 
-Archival is supported in [Global Namespaces](/namespaces#global-namespace) (Namespaces that span multiple clusters).
-When Archival is running in a Global Namespace, it first runs on the active cluster; later it runs on the standby cluster. Before archiving, a history check is done to see what has been previously archived.
+Archival is supported in [Multi-region Namespaces](/namespaces#multi-region-namespace) (Namespaces that span multiple clusters).
+When Archival is running in a Multi-region Namespace, it first runs on the active cluster; later it runs on the standby cluster. Before archiving, a history check is done to see what has been previously archived.
```

Comment:
```suggestion
When Archival is running in a Multi-region Namespace, it first runs on the active Temporal Cluster; later it runs on the standby Temporal Cluster.
Before archiving, a history check is done to see what has been previously archived.
```

Reply:
Change made as suggested, "Archival.mdx" file audited for multiple sentences on a single line. Also minor improvements near the wrapped lines as found.